policies/animations.md

```diff
@@ -1,86 +1,92 @@
-Allowed Animations Policy
+Allow Layout Animations Policy
 ===========
 
-The 'animations' policy-controlled feature can be used in a document or frame to
-restrict the set of CSS properties which can be animated. This can ensure smooth
-animations, by only allowing those properties which can be animated on the GPU,
-using the hardware compositor.
+The 'layout-animations' policy-controlled feature can be used in a document or frame to
+restrict the set of CSS properties which can be animated. This feature is proposed to 
+ensure certain types of animations which *may* lead to layout and potentially heavy CPU
+load are blocked.
 
 What does that mean?
 ------------
 
-In order to produce animations on the web, developers declare transitions inside
-of CSS. These transitions can be used to animate just about any CSS property --
-backgrounds, sizes, margins, fonts, borders, positions -- there's almost no
-limit. In order to make this happen, the browser has to update the values for
-all of these properties, and then calculate all of the other effects that has on
-the page, and it has to update the page like this every frame, sixty times per
-second.
-
-If the properties being animated are changing size or position, this can easily
-cause many other items on the page to have to be moved around constantly, and
-browsers can struggle to keep up. When that happens, the result is a stuttering
-animation and a poorly performing web page, and sometimes even the entire slows
-down, as the animation consumes all available CPU cycles.
-
-One way to avoid this scenario is to move the animations off of the CPU
-entirely. Modern desktop computers and mobile devices have powerful GPUs that
-can perform certain kinds of animations much more efficiently, and can leave the
-CPU free to handle other things, like responding to user input.
-
-Deeper explanation
+In order to produce animations on the web, developers declare transitions of style
+in either CSS (e.g., `@keyframes` and `animation`) or JavaScript (e.g., `element.animate()`). 
+In principal, these transitions can be used to animate just about any CSS property (
```

Comment:
Minor nit -- can you move the "(" to the next line to avoid unnecessary whitespace after it?

Reply:
Done.